docs/blog/2020-02-17-changing-the-oss-status-quo/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Changing the Open Source Status Quo"
+date: 2020-02-17
+author: Aisha Blake
+excerpt: "How can we increase the diversity of open source contributors? As maintainers, we need to listen to a diverse group of community members and make thoughtful changes to create a truly inclusive project."
+tags:
+  - open-source
+  - community
+---
+
+In business and in our individual careers, so many of us rely heavily on open source software. We need it but open source isn't as open as it's often made out to be. Too many projects are unfriendly, even openly hostile, to new contributors. This effect, combined with the intersection of many underrepresented identities in tech effectively shuts people out of the process. Black, indigenous, and people of color (BIPOC) and women are at some of the most visible of these intersections. There are many more. Consider _who_ contributes to open source.
+
+The status quo is made and remade every time we as maintainers add to our documentation. Every time we, as maintainers, engage with the community via social media, issue queues, and our documentation, we help decide who contributes. We contribute to their safety and their productivity. Right now, barriers to contribution are severely limiting our capacity to create and sustain open source projects.
+
+There's an opportunity here. There are thousands of highly skilled potential contributors ready, willing, and able to contribute to open source projects but they need to know that they'll be safe in doing so. A handful of projects are intentionally focusing on this work and making incredible strides towards a more inclusive open source community.
+
+Gatsby's most admirable quality is the company's commitment to that community. Yes, it's a great tool but what use is any tool that is too difficult to use or too obscure for most people to access? One of Gatsby's core values is "You Belong Here". We back it up with a clear code of conduct that we consistently reference and enforce. The documentation is vast, includes specifics around contributing, and provides materials that make it easy for people outside of the core team to teach as well as learn. New contributors who have had a PR merged in are invited to the GitHub organization and offered SWAG as a thank you. There are even a limited number of one-on-one mentorship opportunities available.
```

Comment:
```suggestion
Gatsby's most admirable quality is the company's commitment to that community. Yes, it's a great tool but what use is any tool that is too difficult to use or too obscure for most people to access? One of Gatsby's core values is "You Belong Here". We back it up with a clear code of conduct that we consistently reference and enforce. The documentation is vast, includes specifics around contributing, and provides materials that make it easy for people outside of the core team to teach as well as learn. New contributors who have had a PR merged in are invited to the GitHub organization and offered swag as a thank you. There are even a limited number of one-on-one mentorship opportunities available.
```
Should swag be in all-caps?

Reply:
It's short for "stuff we all get"!

Reply:
I'm taking the suggestion for consistency with other posts and docs but let the record show that I still think it _should_ be in all caps! 😂